API/Data/Repositories/ReadingListRepository.cs

```diff
@@ -241,8 +242,11 @@ public async Task<IEnumerable<ReadingListItemDto>> AddReadingProgressModifiers(i
 
         foreach (var item in items)
         {
-            var progress = userProgress.Where(p => p.ChapterId == item.ChapterId);
-            item.PagesRead = progress.Sum(p => p.PagesRead);
+            var progress = userProgress.SingleOrDefault(p => p.ChapterId == item.ChapterId);
+            if (progress == null) continue;
```

Comment:
You can skip the null check and use null coalescing instead. 

item.ProgressLastModifiedutc = progress?.LastModifiedUtc;

Add the non Utc field to be consistent.

Reply:
ok